### PR TITLE
feat: add article page

### PR DESCRIPTION
**Pull request by limsohee1002**
### Description
#12 Articles - Image module
#21 Articles Markdown styling
#11  Articles Header (Edit Side Nav + On this Page + Breadcrumbs) <- sidenav I didn't work on it yet.
#13 Articles - Rating module
#20 Articles - Related articles module
Add category page
Add section page

### Type(s) of changes

- [ ] Bug fix
- [x] New feature
- [ ] Update to an existing feature

### Applicable screenshots
https://www.loom.com/share/1fddc30664d942a79e3cbcc074c29dae?sid=1f72dce9-69f3-41b9-90da-a7c050407ef0

### CODE REVIEW

File: src/modules/article-page/ArticlePageModule.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import { ArticlePageData } from '../../lib/types';
+import cn from 'classnames';
+
+type Props = {
+  articlePageData: ArticlePageData;
+};
+
+export const ArticleBreadcrumbs: FC<Props> = ({ articlePageData }) => {
```
Comment (mokaymokay) [inline src/modules/article-page/ArticlePageModule.tsx:9]: I'm not seeing breadcrumbs in the preview.. is this data available on the page? why isn't it shown?

**Review by mokaymokay — APPROVED** [review]

File: src/modules/article-page/ArticlePageModule.tsx (hunk 1 above)
Comment (limsohee1002) [reply src/modules/article-page/ArticlePageModule.tsx:9]: oh that is bug I missed that! I will update it!

File: src/modules/homepage/HomepageModule.tsx [hunk 2]
```diff
@@ -162,16 +167,18 @@ const ArticleLinkCard: FC<{ title: string; description: string; url: string }> =
...
       target="_self"
     >
-      <div className="absolute top-0 right-0 transition z-10 py-padding-small px-padding-medium opacity-0 group-hover:opacity-100">
+      <div className="flex flex-col">
+        <h4 className="transition subheading-2 text-light-neutral-1 dark:text-dark-neutral-1 group-hover:text-light-pink-vibrant dark:group-hover:text-dark-pink-vibrant">
+          {title}
+        </h4>
+        <p className="body-3 text-light-neutral-2 dark:text-dark-neutral-2">{description}</p>
+      </div>
+      <div className="transition opacity-0 group-hover:opacity-100">
```
Comment (nahbee10) [inline src/modules/homepage/HomepageModule.tsx:179]: this creates a bit of regression on the short description items: 
Screenshot 2024-11-02 at 11 51 25 AM

**Review by nahbee10 — CHANGES_REQUESTED** [review]
- did you also add image module in this PR? Images are rendered in this way, without any vertical paddings and the gray background(with icon+caption) as [the design](https://www.figma.com/design/QkBERlCkBKBAWRqELCC8uq/%5BUniswap%5D-%F0%9F%8D%84-Spore-Library-w%2F-Web?m=auto&node-id=5046-18654)
Screenshot 2024-11-02 at 11 49 48 AM

**Comment by nahbee10** [comment]
- also noticed this bug on the theme switcher on mobile. will add this to the QA items: 
Screenshot 2024-11-02 at 11 46 29 AM

**Comment by nahbee10** [comment]
> * did you also add image module in this PR? Images are rendered in this way, without any vertical paddings and the gray background(with icon+caption) as [the design](https://www.figma.com/design/QkBERlCkBKBAWRqELCC8uq/%5BUniswap%5D-%F0%9F%8D%84-Spore-Library-w%2F-Web?m=auto&node-id=5046-18654)
> 
https://uniswaplabs.zendesk.com/auth/v2/login/signin?return_to=https%3A%2F%2Funiswaplabs1730152595.zendesk.com%2Fhc%2Fen-us%2Farticles%2F31449357746573-What-are-these-sections-and-articles-doing-here&theme=hc&locale=en-us&brand_id=31433470015629&auth_origin=31433470015629%2Cfalse%2Ctrue

**Comment by limsohee1002** [comment]
image
this is image on 
https://uniswaplabs.zendesk.com/hc/en-us/articles/17674084283149-How-to-swap-on-UniswapX
this article. The caption is part of the image, and we can't really render it separately...
https://uniswaplabs.zendesk.com/hc/article_attachments/24477317213581

**Comment by limsohee1002** [comment]
> * did you also add image module in this PR? Images are rendered in this way, without any vertical paddings and the gray background(with icon+caption) as [the design](https://www.figma.com/design/QkBERlCkBKBAWRqELCC8uq/%5BUniswap%5D-%F0%9F%8D%84-Spore-Library-w%2F-Web?m=auto&node-id=5046-18654)
> 
> Screenshot 2024-11-02 at 11 49 48 AM

The link on the image doesn't seem to work anymore...
https://uniswaplabs.zendesk.com/hc/en-us/articles/31449357746573-What-are-these-sections-and-articles-doing-here

**Comment by limsohee1002** [comment]
Updated preview!:
https://www.loom.com/share/89b25eede3fe4e14b425ecccf0d7d45d?sid=af11d0de-3b83-42e0-9989-ab8dc0d6cdf3

**Pull request merged**